server/server.js

```diff
@@ -0,0 +1,9 @@
+var express = require("express");
```

Comment:
On second thoughts, since the latest meeting with the client representative and product owner we can safely assume we will be using both a display page and deployed server - we can keep these in for future development.